config/development.json

```diff
@@ -137,5 +138,6 @@
 		"vip/deploys": true,
 		"vip/logs": true,
 		"vip/support": true
-	}
+	},
+	"signup_theme_demos": true
```

Comment:
Derp. That was supposed to go only inside of `features` like in the other files. And this is also the old name, before I switched to `/` and `-` as separators and amended my commit.